maggy/util.py

```diff
@@ -275,7 +275,6 @@ def register_environment(app_id, run_id):
     set_ml_id(app_id, run_id)
     # Create experiment directory.
     EnvSing.get_instance().create_experiment_dir(app_id, run_id)
-    tensorboard._register(EnvSing.get_instance().get_logdir(app_id, run_id))
```

Comment:
Not sure why this got removed

Reply:
I think I removed it in an effort to keep tf from importing when using PyTorch DDP only. But honestly that seems impossible with the way it is intertwined with everything at the moment. I'm gonna put it back in.